usaspending_api/broker/management/sql/create_business_categories_functions.sql

```diff
@@ -1,16 +1,162 @@
+    the_ability_one_program     boolean,
+    dot_certified_disadvantage  boolean,
+    emerging_small_business     boolean,
+    federally_funded_research   boolean,
+    historically_underutilized  boolean,
+    labor_surplus_area_firm     boolean,
+    sba_certified_8_a_joint_ve  boolean,
+    self_certified_small_disad  boolean,
+    small_agricultural_coopera  boolean,
+    small_disadvantaged_busine  boolean,
+    community_developed_corpor  boolean,
+    domestic_or_foreign_entity  text,
+    foreign_owned_and_located   boolean,
+    foreign_government          boolean,
+    international_organization  boolean,
+    domestic_shelter            boolean,
+    hospital_flag               boolean,
+    veterinary_hospital         boolean,
+
+    -- NON-PROFIT
+    foundation                  boolean,
+    community_development_corp  boolean,
+    nonprofit_organization      boolean,
+
+    -- HIGHER EDUCATION
+    educational_institution     boolean,
+    other_not_for_profit_organ  boolean,
+    state_controlled_instituti  boolean,
+    c1862_land_grant_college    boolean,
+    c1890_land_grant_college    boolean,
+    c1994_land_grant_college    boolean,
+    private_university_or_coll  boolean,
+    minority_institution        boolean,
+    historically_black_college  boolean,
+    tribal_college              boolean,
+    alaskan_native_servicing_i  boolean,
+    native_hawaiian_servicing   boolean,
+    hispanic_servicing_institu  boolean,
+    school_of_forestry          boolean,
+    veterinary_college          boolean,
+
+    -- GOVERNMENT
+    us_federal_government       boolean,
+    federal_agency              boolean,
+    us_government_entity        boolean,
+    interstate_entity           boolean,
+    us_state_government         boolean,
+    council_of_governments      boolean,
+    city_local_government       boolean,
+    county_local_government     boolean,
+    inter_municipal_local_gove  boolean,
+    municipality_local_governm  boolean,
+    township_local_government   boolean,
+    us_local_government         boolean,
+    local_government_owned      boolean,
+    school_district_local_gove  boolean,
+    us_tribal_government        boolean,
+    indian_tribe_federally_rec  boolean,
+    housing_authorities_public  boolean,
+    airport_authority           boolean,
+    port_authority              boolean,
+    transit_authority           boolean,
+    planning_commission         boolean
+)
 returns text[] as $$
 declare
     bc_arr text[];
 begin
 
-    -- SMALL BUSINESS
-    if small_business_competitive IS TRUE or for_profit_organization IS TRUE
+-- BUSINESS (FOR-PROFIT ORGANIZATION)
+    if contracting_officers_deter = 'S' or women_owned_small_business IS TRUE or economically_disadvantaged IS TRUE or joint_venture_women_owned IS TRUE or emerging_small_business IS TRUE or self_certified_small_disad IS TRUE or small_agricultural_coopera IS TRUE or small_disadvantaged_busine IS TRUE
```

Comment:
Can this be split into multiple lines for readability?

Reply:
Fixed in latest commit.